dedupe/ecr_to_fhir_conversion/ECR to FHIR setup.ipynb

```diff
@@ -0,0 +1,196 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3648cc77-4b5c-4383-921a-0a48942ff0d2",
+   "metadata": {},
+   "source": [
+    "#### 0. Import necessary libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "5e22daca-cb6a-4619-9964-b4a75b903a2c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from lxml import etree\n",
+    "from pathlib import Path\n",
+    "from zipfile import ZipFile"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "729f3800-cf10-45ce-b072-44ee968b95de",
+   "metadata": {},
+   "source": [
+    "#### 1. Extract all of the zip files from the `LAC_DATA` zip archive"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "b356a1b9-99d0-45e4-9e87-85ac7faa0b2f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "infile = Path(\"./LAC_DATA.zip\")\n",
+    "outfile = Path(\"./LAC_DATA\")\n",
+    "with ZipFile(infile, \"r\") as zip_file:\n",
+    "    zip_file.extractall(path=\".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e95940d-9fcd-4629-8660-2389a5db00fd",
+   "metadata": {},
+   "source": [
+    "#### 2. For each zip file in the archive, extract all of the other files into a folder named after the zip file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "6e846502-92c4-4b98-bcbd-86a094f2feab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_files = list(outfile.glob(\"*.*\"))\n",
+    "for f in all_files:\n",
+    "    with ZipFile(f, \"r\") as zip_file:\n",
+    "        # assumes you are in the phdi directory\n",
+    "        zip_file.extractall(path=f\"./containers/ecr-viewer/seed-scripts/baseECR/{f.stem}/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c47e3df2-631b-4ce6-aa11-f390ad783ef2",
+   "metadata": {},
+   "source": [
+    "#### 3. Launch FHIR Conversion Docker container\n",
+    "\n",
+    "Navigate to the `ecr-viewer` subdirectory in the `phdi` directory via `cd containers/ecr-viewer` in your terminal and run the following commands:\n",
+    "\n",
+    "```bash\n",
+    "dockId=$(docker run --rm -d -it -p 8080:8080 \"$(docker build -q ./../fhir-converter/)\")\n",
+    "echo $dockId\n",
+    "```\n",
+    "\n",
+    "As long as you see a value printed to the screen, your container should be up and running."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3f043ac-4bac-4a1a-9f30-efe4f7b1b0b2",
+   "metadata": {},
+   "source": [
+    "#### 4. Loop through each folder and extract the files\n",
+    "\n",
+    "Run the following for loop in your terminal:\n",
+    "\n",
+    "```bash\n",
+    "for d in ./baseECR/* ; do\n",
+    "    #first escape \", then /, and finally remove all new lines\n",
+    "    rr=$(sed -e 's/\"/\\\\\"/g ; s=/=\\\\\\/=g ; $!s/$/\\\\n/' \"$d/CDA_RR.xml\" | tr -d '\\n')\n",
+    "    eicr=$(sed 's/\"/\\\\\"/g ; s=/=\\\\\\/=g ; $!s/$/\\\\n/'  \"$d/CDA_eICR.xml\" | tr -d '\\n')\n",
```

Comment:
With the current version of the fhir-converter API, I need to replace the `\t` characters.  This could be just my setup, but I wanted to double-check that on your machine the fhir-converter API works as expected without this step?

Reply:
I hit an issue during testing on my Windows machine, and now I'm away from home working on my laptop. I'm going to put this PR into draft status until I can confirm that escaping `\t` is or isn't necessary. Thanks for your diligence on this!